docs/tutorials/misc/custom-envelope.mdx

```diff
@@ -155,4 +155,7 @@ with the new SDK version.
 ## Conclusion
 Using GitHub actions, creating and building custom envelopes is simple and doesn't
 require any special setup. Once a variant is created, a few clicks are all that's
-needed to build the envelope.
+needed to build the envelope.import ScreenBuildActionForm from "../../_images/envelope-build-action-form.png";
```

Comment:
Did this go in?

Reply:
Unfortunately yes... :(
https://github.com/toitware/web-docs/pull/507